data_steward/vocabulary.py

```diff
@@ -1,11 +1,179 @@
-CONCEPT = 'concept'
-CONCEPT_ANCESTOR = 'concept_ancestor'
-CONCEPT_CLASS = 'concept_class'
-CONCEPT_RELATIONSHIP = 'concept_relationship'
-CONCEPT_SYNONYM = 'concept_synonym'
-DOMAIN = 'domain'
-DRUG_STRENGTH = 'drug_strength'
-RELATIONSHIP = 'relationship'
-VOCABULARY = 'vocabulary'
-VOCABULARY_TABLES = [CONCEPT, CONCEPT_ANCESTOR, CONCEPT_CLASS, CONCEPT_RELATIONSHIP, CONCEPT_SYNONYM, DOMAIN,
-                     DRUG_STRENGTH, RELATIONSHIP, VOCABULARY]
+import os
+import csv
```

Comment:
I would move `import csv` before `import os`.  While not technically required to be in alphabetic order in the import groups, I think it's easier to peruse the list later when maintaining the module.